tests/main/lxd/prep-snapd-in-lxd.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
```

Comment:
Nitpick, use sh when the extension is .sh and we don't need bash features.

Reply:
sure